packages/belos/epetra/example/GCRODR/PrecGCRODREpetraExFile.cpp

```diff
@@ -154,12 +154,18 @@ bool proc_verbose = false;
   Teuchos::RCP<Ifpack_CrsRiluk> ilukFactors;
   //
   ilukGraph = Teuchos::rcp(new Ifpack_IlukGraph(A->Graph(), Lfill, Overlap));
-  int info = ilukGraph->ConstructFilledGraph();
+#ifndef NDEBUG
```

Comment:
Belos has a required dependence on Teuchos, so you could do this:
```c++
#include "Teuchos_Assert.hpp"
// ...
int info = ilukGraph->ConstructFilledGraph();
TEUCHOS_ASSERT( info == 0 );
```
This approach would make the most sense for tests and examples, where we always want to check the assertion.  ("Tests" that only use `assert` aren't actually testing those assertions in any PR test right now.)

Reply:
OK. I'll switch them over.
"tests and examples"? Would using `TEUCHOS_ASSERT` also be preferred for `packages/belos/epetra/src/BelosEpetraUtils.cpp`?
Or shall I just leave that one be with the assert & macro lines?

Reply:
@ZUUL42 Thanks!  "Tests and examples" means "not library source code."  `TEUCHOS_ASSERT` throws an exception; that's different than what `assert` does, and we don't want to change the library's behavior.

> Would using `TEUCHOS_ASSERT` also be preferred for `packages/belos/epetra/src/BelosEpetraUtils.cpp`?

That's library source code; it would be best to leave that alone.